### PR TITLE
[KYUUBI #3646][UI] Init Session Statistic Page

### LINKED ISSUE #3646: [FEATURE] [UI] Add session statistic page
### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### Search before asking

- [X] I have searched in the [issues](https://github.com/apache/incubator-kyuubi/issues?q=is%3Aissue) and found no similar issues.

### Describe the feature

Add session statistic UI page

### Motivation

_No response_

### Describe the solution

_No response_

### Additional context

_No response_

### Are you willing to submit PR?

- [ ] Yes. I can submit a PR independently to improve.
- [X] Yes. I would be willing to submit a PR with guidance from the Kyuubi community to improve.
- [ ] No. I cannot submit a PR at this time.

### PR DESCRIPTION
**Pull request by zwangsheng**
### _Why are the changes needed?_

Init Session Statistic Page

Close #3646

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.readthedocs.io/en/master/develop_tools/testing.html#running-tests) locally before make a pull request

popo_2023-03-20  17-47-24

### CODE REVIEW

**Comment by zwangsheng** [comment]
Gentle ping @zhaohehuhu, do you have any suggestions and ideas about front-end unit test?

File: kyuubi-server/web-ui/package.json [hunk 1]
```diff
@@ -17,6 +17,7 @@
   "dependencies": {
     "@element-plus/icons-vue": "^2.0.9",
     "axios": "^0.27.2",
+    "date-fns": "^2.29.3",
```
Comment (pan3793) [inline kyuubi-server/web-ui/package.json:20]: LICENSE-binary should be updated when introduce new deps

File: kyuubi-server/web-ui/src/views/session/session-statistics/index.vue [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+      })
+  }
+  const getList = () => {
+    _getList(getAllSessions)
+  }
+  getList()
+</script>
+
+<style scoped lang="scss">
+  header {
```
Comment (pan3793) [inline kyuubi-server/web-ui/src/views/session/session-statistics/index.vue:115]: where is the reference

File: kyuubi-server/web-ui/src/views/session/session-statistics/index.vue [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+  </el-card>
+</template>
+
+<script lang="ts" setup>
+  import { format } from 'date-fns'
+  import { getAllSessions, deleteSession } from '@/api/session'
+  import { ElMessage } from 'element-plus'
+  import { useI18n } from 'vue-i18n'
+  import { useTable } from '@/views/common/use-table'
+  const { t } = useI18n()
```
Comment (pan3793) [inline kyuubi-server/web-ui/src/views/session/session-statistics/index.vue:87]: why name it `t`?

File: kyuubi-server/web-ui/src/views/session/session-statistics/index.vue [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+              <span>
+                <el-tooltip
+                  effect="dark"
+                  :content="$t('delete')"
+                  placement="top"
+                >
+                  <template #default>
+                    <el-button type="danger" icon="Delete" circle />
+                  </template> </el-tooltip
+              ></span>
```
Comment (pan3793) [inline kyuubi-server/web-ui/src/views/session/session-statistics/index.vue:72]: strange style

File: kyuubi-server/web-ui/src/views/session/session-statistics/index.vue (hunk 3 above)
Comment (zwangsheng) [reply kyuubi-server/web-ui/src/views/session/session-statistics/index.vue:83]: To remain similar to the `$t` usage of i18n above

File: kyuubi-server/web-ui/src/views/session/session-statistics/index.vue (hunk 4 above)
Comment (zwangsheng) [reply kyuubi-server/web-ui/src/views/session/session-statistics/index.vue:68]: due to `  "jsxBracketSameLine": false` in `.prettierrc` config, but +1 for bracket same line

Comment (zwangsheng) [reply kyuubi-server/web-ui/src/views/session/session-statistics/index.vue:68]: You can review https://github.com/apache/kyuubi/pull/4571 first, and i will rebase this PR.

Comment (pan3793) [reply kyuubi-server/web-ui/src/views/session/session-statistics/index.vue:68]: > You can review #4571 first, and i will rebase this PR.

done

**Review by pan3793 — APPROVED** [review]

**Comment by pan3793** [comment]
Thanks, merged to master, and also credits to the original author @zhaohehuhu

**Pull request closed without merging**